plinth/templates/service.html

```diff
@@ -68,4 +68,18 @@
     </form>
   {% endblock %}
 
+  {% block uninstall %}
+    {% if diagnostics_module_name and service.allow_uninstall %}
+      <br>
+      <p>
+        <a class="btn btn-default btn-md"
+           href="{% url 'uninstall' diagnostics_module_name %}">
```

Comment:
We could later create a task to rename diagnostic_module_name to app_name and automatically infer it from Django application name.